README.md

```diff
@@ -11,168 +13,170 @@ GodTools
+#### Additional Resources:
+- Solid principles: 
+    - https://www.geeksforgeeks.org/solid-principle-in-programming-understand-with-real-life-examples/
+- Coordinator (Flow.swift): 
+    - https://khanlou.com/2015/01/the-coordinator/
+    - https://twittemb.github.io/posts/2017-11-08-RxFlow-Part1/
+    - https://twittemb.github.io/posts/2017-12-09-RxFlow-Part2/
 
-##### ViewModels
+#### Fastlane
 
-ViewModels are considered a view representation.  They're the data backing of a view.
+Below are some helpful references to GitHub Actions Workflows and Fastlane Files that the GodTools project uses.
 
-###### ViewModel Responsibilities:
-- Provides data outputs to the View which the View can observe and react to.
-- Each data output connects to a UseCase (Domain Layer) which computes data.  Computing can also happen directly in the ViewModel although UseCases are preferred.
+- Project Fastlane Fastfile (https://github.com/CruGlobal/godtools-swift/blob/develop/fastlane/Fastfile) points to shared Fastfile (https://github.com/CruGlobal/cru-fastlane-files/blob/master/Fastfile)
 
-#### Domain Layer
+- Uses a combination of Fastlane Environment Variables(https://github.com/CruGlobal/godtools-swift/blob/develop/fastlane/.env.default) and GitHub Secrets
 
-The domain layer is responsible for the business rules of the application and is broken up into Use Cases.  The benefits of the domain layer include:
-- It avoids code duplication.
-- It improves readability in ViewModels that use domain layer use cases.
-- It improves testability of the app.
-- It avoids large ViewModels by allowing you to split responsibilities.
+- GitHub Actions Build Workflow: https://github.com/CruGlobal/godtools-swift/blob/develop/.github/workflows/build.yml
 
+- GitHub Actions OneSky Workflow: https://github.com/CruGlobal/godtools-swift/blob/develop/.github/workflows/download_onesky_translations.yml
 
-##### - Use Cases
+- GitHub Actions OneSky Workflow Dependency Plugin: https://github.com/thekie/fastlane-plugin-onesky
 
-Use cases make up the domain layer.  Use cases should be responsible for a single task and named to reflect that task.  For example: AuthenticateUserUseCase, GetSpotlightToolsUseCase.
+#### Conventions
 
-UseCase Best Practices:
+- [Classes](#classes)
+- [Project Folder Structure](#project-folder-structure)
 
-- Should have a single exposed method (public, internal) that takes zero or more inputs and produces a single output that is an AnyPublisher. 
-- UseCases can have private methods, however, as we move to dependency inversion I think private methods will become less and less.
-- Inputs should not be publisher types. Instead the ViewModel should react to changes which then triggers the UseCase.
-- Should not depend on UseCases. If specific Output from a UseCase is needed, instead inject that Output as an Input from the ViewModel.
```

Comment:
This line is a little confusing.  Maybe "If specific Output from a _different_ UseCase is needed,  ..."